src/USER-MISC/compute_pressure_cylinder.cpp

```diff
@@ -0,0 +1,493 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#include "math.h"
+#include "string.h"
+#include "stdlib.h"
+#include "compute_pressure_cylinder.h"
+#include "atom.h"
+#include "update.h"
+#include "force.h"
+#include "pair.h"
+#include "neighbor.h"
+#include "neigh_request.h"
+#include "neigh_list.h"
+#include "group.h"
+#include "memory.h"
+#include "error.h"
+
+using namespace LAMMPS_NS;
+
+static const char cite_compute_pressure_cylinder[] =
+  "compute pressure/cylinder:\n\n"
+  "@Article{Addington,\n"
+  " author = {C. K. Addington, Y. Long, K. E. Gubbins},\n"
+  " title = {The pressure in interfaces having cylindrical geometry},\n"
+  " journal = {J.~Chem.~Phys.},\n"
+  " year =    2018,\n"
+  " volume =  149,\n"
+  " pages =   {084109}\n"
+  "}\n\n";
+
+if (lmp->citeme) lmp->citeme->add(cite_compute_pressure_cylinder);
```

Comment:
to use this feature, you need to include the citeme.h header file. without, compilation is failing (see the failed tests on the pull request).